sandbox/grommet-app/src/pages/layouts/kinds/Insider/Insider.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import { 
+    Box,
+    dark,
+    Page, 
+    PageContent, 
+    ThemeContext,
+} from 'grommet';
+import { Hero } from './Hero';
+import { Stories } from './Stories';
+
+export const Insider = () => {
+    const theme = React.useContext(ThemeContext);
+    const { global } = theme;
+    const { colors } = global;
+    const { background } = colors;
+
+    console.log('v2 theme - theme', theme)
+    // console.log('v2 theme - global', global)
+    // console.log('v2 theme - colors', colors)
+
+    const adjustedTheme = {
```

Comment:
I was running into an issue with the theme on primary buttons, where it seemed like grommet was trying to be too "smart" about the dark vs light choice of text/icon color.

I did a workaround for the text color via button.extend in my PR, but clearly missed covering the icon case:

<img width="417" alt="Screenshot 2025-05-07 at 2 44 22 PM" src="https://github.com/user-attachments/assets/b1b55ab3-ee00-4931-a3be-bb0fb2a23ee9" />

Regardless, something I need to just investigate a little further.

Reply:
For some reason it only happens for the rest state, but then "hover" it gets fixed.